static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -154,19 +161,18 @@ const bootCommercial = async (): Promise<void> => {
 	try {
 		await loadFrontendBundle();
 		await loadDcrBundle();
-		await loadModules();
 
-		return catchErrorsWithContext(
-			[
-				[
-					'ga-user-timing-commercial-end',
-					function runTrackPerformance(): void {
-						EventTimer.get().trigger('commercialEnd');
-					},
-				],
-			],
-			tags,
-		);
+		const allModules: Array<Parameters<typeof loadModules>> = [
+			[commercialBaseModules, 'commercialBaseModulesLoaded'],
+			[commercialExtraModules, 'commercialExtraModulesLoaded'],
+		];
+		const promises = allModules.map((args) => {
+			return loadModules(...args);
+		});
+
+		await Promise.all(promises).then(() => {
+			EventTimer.get().trigger('commercialEnd');
+		});
```

Comment:
<details>
<summary>Stylistic, feel free to ignore.</summary>

If you want, you can simplify the return:

```suggestion
const promises = allModules.map((args) => loadModules(...args);

await Promise.all(promises).then(() => {
EventTimer.get().trigger('commercialEnd');
});
```

Or even drop the declaration of `promises` altogether.

</details>